compiler/rustc_mir/src/borrow_check/mod.rs

```diff
@@ -643,6 +643,8 @@ impl<'cx, 'tcx> dataflow::ResultsVisitor<'cx, 'tcx> for MirBorrowckCtxt<'cx, 'tc
                     self.consume_operand(location, (input, span), flow_state);
                 }
             }
+
+            StatementKind::CopyNonOverlapping(..) => todo!(),
```

Comment:
I think you can initially just consider this to behave exactly the same as a regular call. As in, just `consume_operand` the three arguments.